backend/src/main/java/com/bakdata/conquery/models/query/preview/EntityPreviewExecution.java

```diff
@@ -154,13 +154,16 @@ private List<EntityPreviewStatus.TimeStratifiedInfos> toChronoInfos(PreviewConfi
 							   .collect(Collectors.toMap(PreviewConfig.InfoCardSelect::select, Function.identity()));
 
 			// Group lines by year and quarter.
-			final List<EntityPreviewStatus.YearEntry> yearEntries = createYearEntries(entityResult, query.getResultInfos(), printSettings, select2desc);
+			final Function<Object[], Map<String, Object>> lineTransformer = createLineTransformer(query.getResultInfos(), select2desc, printSettings);
```

Comment:
```suggestion
			final Function<Object[], Map<String, Object>> lineTransformer = createLineToMapTransformer(query.getResultInfos(), select2desc, printSettings);
```
Vielleicht ein besserer Name. Ist mir aufgefallen weil ich erstmal nachschauen musste, was hier eigentlich transformiert wird